CMakeLists.txt

```
@@ -338,19 +338,19 @@ if(NOT "${BASH}" STREQUAL "BASH-NOTFOUND" AND GINKGO_DEVEL_TOOLS)
     find_program(GIT git)
     if(NOT "${GIT}" STREQUAL "GIT-NOTFOUND")
         add_custom_target(format_header
-            COMMAND echo "format header on the modified code files except build/examples/third_party/ginkgo.hpp"
+            COMMAND echo "format header on the modified code files except build/examples/third_party/accessor/dev_tools/ginkgo.hpp"
             COMMAND bash -c "git diff --name-only origin/develop...HEAD | \
-                grep -Ev 'build|examples|third_party|ginkgo.hpp' | \
+                grep -Ev 'build|examples|third_party|accessor/|dev_tools/|ginkgo.hpp' | \
```

nit: are the `/` necessary?
```suggestion
                grep -Ev 'build|examples|third_party|accessor|dev_tools|ginkgo.hpp' | \
```